packages/authentication/addon/templates/components/cardstack-authentication-launcher.hbs

```diff
@@ -14,7 +19,7 @@
   {{/if}}
 
   {{#if (is-component "cardstack-workflow")}}
-    <div class="cardstack-rail-item {{if isOpen 'active'}}">
+    <div class={{concat "cardstack-rail-item " (if isOpen "active")}}>
```

Comment:
why did you need `concat` here? this generally should work fine without it

Reply:
Because we can only have double quotes in templates (a linting rule) and I can't embed a double-quoted string inside class="...".

Reply:
hbs```
<div class="cardstack-rail-item {{if isOpen "active"}}">
```
does ☝️ not work?

Reply:
I'll try that in the linter PR, thank you.